seedDB/Update-Seed-Database.md

```diff
@@ -0,0 +1,23 @@
+# Update cBioPortal seed database files 
+To update the seed database files to a recent version you should follow these steps:
```

Comment:
maybe to make it very clear that this is documentation for developers intending to update the files in github, start with something like:
"For developers: to update the seed database files in this github repository to a new version you should follow these steps:"